examples/plot_fpca.py

```diff
@@ -0,0 +1,113 @@
+basis_fd = fd.to_basis(basis)
+basis_fd.plot()
+
+##############################################################################
+# We initialize the FPCABasis object and run the fit function to obtain the
+# first 2 principal components. By default the principal components are
+# expressed in the same basis as the data. We can see that the obtained result
+# is similar to the discretized case.
+fpca = FPCABasis(n_components=2)
+fpca.fit(basis_fd)
+fpca.components.plot()
+
+##############################################################################
+# To better illustrate the effects of the obtained two principal components,
+# we add and subtract a multiple of the components to the mean function.
+# As the module modifies the original data, we have to fetch the data again.
+# And then we get the mean function and plot it.
+dataset = fetch_growth()
+fd = dataset['data']
+basis_fd = fd.to_basis(BSpline(n_basis=7))
+mean_fd = basis_fd.mean()
+mean_fd.plot()
+
+##############################################################################
+# Now we add and subtract a multiple of the first principal component. We can
+# then observe now that this principal component represents the variation in
+# growth between the children.
+mean_fd.coefficients = np.vstack([mean_fd.coefficients,
+                                  mean_fd.coefficients[0, :] +
+                                  20 * fpca.components.coefficients[0, :]])
+mean_fd.coefficients = np.vstack([mean_fd.coefficients,
+                                  mean_fd.coefficients[0, :] -
+                                  20 * fpca.components.coefficients[0, :]])
+mean_fd.plot()
+
+##############################################################################
+# The second component is more interesting. The most appropriate explanation is
+# that it represents the differences between girls and boys. Girls tend to grow
+# faster at an early age and boys tend to start puberty later, therefore, their
+# growth is more significant later. Girls also stop growing early
+mean_fd = basis_fd.mean()
+mean_fd.coefficients = np.vstack([mean_fd.coefficients,
+                                  mean_fd.coefficients[0, :] +
+                                  20 * fpca.components.coefficients[1, :]])
+mean_fd.coefficients = np.vstack([mean_fd.coefficients,
+                                  mean_fd.coefficients[0, :] -
+                                  20 * fpca.components.coefficients[1, :]])
+mean_fd.plot()
+
+##############################################################################
+# We can also specify another basis for the principal components as argument
+# when creating the FPCABasis object. For example, if we use the Fourier basis
+# for the obtained principal components we can see that the components are
+# periodic. This example is only to illustrate the effect. In this dataset, as
+# the functions are not periodic it does not make sense to use the Fourier basis
```

Comment:
Then, why not using monomials as the other example?

Reply:
This is to illustrate that we can change the basis but we have to be careful about it.
It is a good idea to further illustrate the change of basis with a monomial basis and show that the effect is much better.

Reply:
Ill use monomials as another example and illustrate it, didn't get your point the other day.